src/components/backend-ai-credential-view.ts

```diff
@@ -77,9 +76,16 @@ export default class BackendAICredentialView extends BackendAIPage {
   @property({type: String}) _defaultFileName = '';
   @property({type: Number}) selectAreaHeight;
   @property({type: Boolean}) enableSessionLifetime = false;
+  @state() private all_vfolder_hosts;
+  @state() private allowed_vfolder_hosts;
```

Comment:
This state variable doesn't seem to be used anywhere.

Reply:
It's used here. https://github.com/lablup/backend.ai-webui/pull/1327/files#diff-1ac42de48a37ca285d9aa6edb95c2cbe64c8474285dee56d6d036008893842d8R381